pjmedia/src/pjmedia/conference.c

```diff
@@ -551,6 +551,7 @@ PJ_DEF(pj_status_t) pjmedia_conf_create( pj_pool_t *pool,
     const pj_str_t name = { "Conf", 4 };
     pj_status_t status;
 
+    PJ_ASSERT_RETURN(samples_per_frame > 0, PJ_EINVAL);
```

Comment:
If we're adding checking, I'm curious why only samples_per_frame is checked

Reply:
a error "Division by zero" report in this line:
https://github.com/pjsip/pjproject/blob/2c4bf48fb5891a3bb1fff90932f40d69d94b8d75/pjmedia/src/pjmedia/conference.c#L2102

`conf->samples_per_frame`  assign in pjmedia_conf_create()

Add the assert  just to emphasize that this will not happen